rmrk-token/tests/utils.rs

```diff
@@ -0,0 +1,437 @@
+use base_io::{BaseAction, EquippableList, FixedPart, InitBase, Part, SlotPart};
+use codec::Encode;
+use gstd::{BTreeMap, BTreeSet};
+use gtest::{Program, RunResult, System};
+use resource_io::Resource;
+use rmrk_io::*;
+use types::primitives::{CollectionAndToken, CollectionId, PartId, ResourceId, TokenId};
+pub const USERS: &[u64] = &[10, 11, 12, 13];
+pub const ZERO_ID: u64 = 0;
+pub const PARENT_NFT_CONTRACT: u64 = 2;
+pub const CHILD_NFT_CONTRACT: u64 = 1;
+
+pub const BASE_ID: u64 = 3;
+pub const CHILD_RESOURCE_ID: ResourceId = 150;
+pub const PARENT_RESOURCE_ID: ResourceId = 151;
+
+pub fn init_rmrk(sys: &System, resource_hash: Option<[u8; 32]>) {
+    sys.init_logger();
+    let rmrk = Program::current(sys);
+    let res = rmrk.send(
+        USERS[0],
+        InitRMRK {
+            name: "RMRKToken".to_string(),
+            symbol: "RMRKSymbol".to_string(),
+            resource_hash,
+            resource_name: "ResourceName".to_string(),
+        },
+    );
+    if resource_hash.is_some() {
+        println!("{:?}", res.decoded_log::<RMRKEvent>());
+        assert!(!res.log().is_empty());
+    } else {
+        assert!(res.log().is_empty());
+    }
+}
+
+pub fn init_base(sys: &System) {
+    let base = Program::from_file(sys, "../target/wasm-projects/release/rmrk_base.wasm");
```

Comment:
```suggestion
    let base = Program::from_file(sys, "../target/wasm32-unknown-unknown/release/rmrk_base.wasm");
```
See the first review comment.